og_format_examples_files/sg558_20240206T000000_R.cdl

```diff
@@ -0,0 +1,549 @@
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:maker = "Sea-Bird Scientific" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:maker_vocabulary = "http://vocab.nerc.ac.uk/collection/L35/current/MAN0013/" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:model = "Unpumped CT sail CTD" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:model_vocabulary = "https://vocab.nerc.ac.uk/collection/L22/current/TOOL1188" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:serial_number = "0363" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:calibration_date = "" ;
+	int64 INSTRUMENT_DATA\ LOGGERS_sg558 ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:_FillValue = -1L ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:long_name = "Seaglider logger sg558" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:type = "data loggers" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:type_vocabulary = "http://vocab.nerc.ac.uk/collection/L05/current/DLOG/" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:maker = "Kongsberg Maritime" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:maker_vocabulary = "http://vocab.nerc.ac.uk/collection/L35/current/MAN0015/" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:model = "Seaglider M1 Glider data logger" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:model_vocabulary = "https://vocab.nerc.ac.uk/collection/L22/current/TOOL1185" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:serial_number = "sg558" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:calibration_date = "" ;
+	string TRAJECTORY ;
+		TRAJECTORY:cf_role = "trajectory_id" ;
+		TRAJECTORY:long_name = "trajectory name" ;
+		TRAJECTORY:data_mode_vocabulary = "" ;
+	string PLATFORM_TYPE ;
+		PLATFORM_TYPE:long_name = "type of glider" ;
+		PLATFORM_TYPE:platform_type_vocabulary = "" ;
+	string PLATFORM_MODEL ;
+		PLATFORM_MODEL:long_name = "model of the glider" ;
+		PLATFORM_MODEL:platform_model_vocabulary = "" ;
+	string WMO_IDENTIFIER ;
+		WMO_IDENTIFIER:long_name = "wmo id" ;
+	string DEPLOYMENT_TIME ;
+		DEPLOYMENT_TIME:long_name = "Date of deployment" ;
+		DEPLOYMENT_TIME:calendar = "gregorian" ;
+		DEPLOYMENT_TIME:standard_name = "time" ;
+		DEPLOYMENT_TIME:units = "seconds since 1970-01-01T00:00:00Z" ;
+		DEPLOYMENT_TIME:axis = "T" ;
+	string DEPLOYMENT_LATITUDE ;
+		DEPLOYMENT_LATITUDE:long_name = "latitude of deployment" ;
+	string DEPLOYMENT_LONGITUDE ;
+		DEPLOYMENT_LONGITUDE:long_name = "longitude of deployment" ;
+
+// global attributes:
+		:geospatial_bounds_crs = "EPSG:4326" ;
+		:geospatial_bounds_vertical_crs = "EPSG:5831" ;
+		:geospatial_lat_min = -63.72288f ;
+		:geospatial_lat_max = -63.34504f ;
+		:geospatial_lon_min = -53.3449f ;
+		:geospatial_lon_max = -51.54048f ;
+		:geospatial_bounds = "POLYGON ((-53.34490203857422, -51.54047775268555, -63.72288131713867, -63.34503936767578))" ;
+		:geospatial_vertical_min = -0.1495176f ;
+		:geospatial_vertical_max = 751.8447f ;
+		:geospatial_vertical_positive = "down" ;
+		:geospatial_vertical_units = "m" ;
+		:time_coverage_start = "2024-02-06T17:05Z" ;
+		:time_coverage_end = "2024-02-23T20:50Z" ;
+		:agency = "UEA" ;
```

Comment:
Victor's latest PR change from using "agency" to "institution" throughout these attrs. I need to update the other example cdls too, so we can ignore this for now

Reply:
ok will update for these examples